inst/builtin/bs5/shiny/_rules.scss

```diff
@@ -70,75 +79,79 @@ $bslib-checkbox-radio-margin-right: 0.35em !default;
-    border-width: 0;
+      // Shadows on top-level cards, not in nested components
+      --bs-card-box-shadow: #{$box-shadow};
+      .bslib-card {
+        --bs-card-box-shadow: none;
+      }
 
-    // We're deliberately avoiding `@include color-mode(dark)` here since
-    // that'd require a fair amount of duplication to properly implement
-    // https://github.com/rstudio/bslib/pull/829/files#r1361262514
-    [data-#{$prefix}theme="dark"] & {
-      box-shadow: $box-shadow-dark;
+      // We're deliberately avoiding `@include color-mode(dark)` here since
+      // that'd require a fair amount of duplication to properly implement
+      // https://github.com/rstudio/bslib/pull/829/files#r1361262514
+      [data-#{$prefix}theme="dark"] & {
+        .bslib-card {
+          --bs-card-box-shadow: none;
+        }
+      }
     }
   }
 
-  // Additional card styling (w/ box shadows in mind)
   .bslib-card {
     .card-header, .card-footer {
       background-color: transparent;
+      font-size: 0.9rem;
     }
     .card-header {
       font-weight: $font-weight-semibold;
-      font-size: 0.9rem;
       line-height: 1.375rem;
     }
     .bslib-sidebar-layout > .sidebar {
       border-top: var(--bslib-sidebar-border);
     }
   }
 
-}
-
-// TODO: move this logic into a "core" bundle that comes with any theme
-// (but you only get the shadows by default in the preset theme)
-@if $bslib-enable-shadows {
-
-  :root {
+  // Set the background color of the main content area
+  .bslib-page-dashboard, .bslib-page-sidebar, .bslib-page-navbar {
     --bslib-dashboard-main-bg: rgba(var(--#{$prefix}emphasis-color-rgb, 0,0,0), 0.03);
-    --bslib-dashboard-sidebar-bg: var(--#{$prefix}body-bg);
-    --bslib-dashboard-sidebar-main-bg: var(--#{$prefix}body-bg);
-  }
 
-  .bslib-page-sidebar, .bslib-page-navbar {
-    .bslib-sidebar-layout {
-      --bslib-sidebar-bg: var(--bslib-dashboard-sidebar-bg);
-      --bslib-sidebar-main-bg: var(--bslib-dashboard-sidebar-main-bg);
+    main, .main, &.main {
+      background-color: var(--bslib-dashboard-main-bg);
+      main, .main, .bslib-sidebar-layout {
+        --bslib-dashboard-main-bg: ; // unset background color rule
+      }
     }
-  }
 
-  .bslib-page-sidebar {
-    --bslib-page-sidebar-title-bg: transparent;
-    --bslib-page-sidebar-title-color: var(--#{$prefix}body-color);
-    > .bslib-sidebar-layout {
+    .bslib-sidebar-layout {
       --bslib-sidebar-main-bg: var(--bslib-dashboard-main-bg);
-      > .main {
-        @include bslib-box-shadows;
-      }
     }
   }
 
-  .bslib-page-navbar {
+  body.bslib-page-dashboard {
+    background-color: var(--bslib-dashboard-main-bg);
+    .bslib-sidebar-layout {
+      --bslib-dashboard-main-bg: ; // unset background color rule
+    }
+  }
 
+  .bslib-page-navbar, .bslib-page-dashboard {
     > .navbar {
       --bslib-navbar-default-bg: transparent;
       --bslib-navbar-inverse-bg: var(--#{$prefix}body-color);
```

Comment:
Should the `border-top:` rule a couple lines below here be using the translucent border?

Reply:
Oh actually we probably don't need that border-too anymore